scratch/app-drive.jsonc

```diff
@@ -0,0 +1,72 @@
+[ // only using array here to have valid in the file. In reality, only the first object here would be returned. Then then the second one would be returned upon fetching from `source`
+
+    {
+        "type": "view",
+        "source": "views/file-list"
+    },
+
+    // Response
+    {
+        "type": "view",
+        "bindings": [
+            {
+                "type": "select",
+                "source": "lookup/available-drives"
+            },
+
+            // Response of lookup, or alternatively embedded right here instead of using a lookup
+            {
+                "type": "select",
```

Comment:
Should we add some options of styling? My point being is not the same a select like this, than the one we would expect on a form.

It could select the styling based on context, but still I wonder if it is worth exploring that.

Reply:
@larkox I'm thinking we should start with traditional selects to begin with, since I think it will fit the situation of most designs, even if it's not exactly what was designed. The more "curated" you make them to fit one design, the less likely it will fit other designs. Adding styling of these can happen as we move forward.